File: movie/src/main/webapp/js/UrlPaths/urlPaths.js

const CONTEXT_PATH      = "/movie";
const SIGNUP            = "/signUp";
const TICKET            = "/ticket";
const ADMIN             = "/admin";

const urlPaths = {
    "LOTTE"             : CONTEXT_PATH + "/lotte",
    "LOGIN"             : CONTEXT_PATH + "/login",
    "LOGOUT"            : CONTEXT_PATH + "/logout",
    "SIGNUP"            : CONTEXT_PATH + SIGNUP,
    "SUCCESS_SIGNUP"    : CONTEXT_PATH + SIGNUP + "/success",
    "CHECK_EMAIL"       : CONTEXT_PATH + SIGNUP + "/checkEmail",
    "CHECK_ID"          : CONTEXT_PATH + SIGNUP + "/checkId",
    "CHECK_PHONE"       : CONTEXT_PATH + SIGNUP + "/checkPhone",
    "PERSON_SEAT"       : CONTEXT_PATH + TICKET + "/personSeat",
    "ORDER_SETTLEMENT"  : CONTEXT_PATH + TICKET + "/orderSettlement",
    "BOOK"              : CONTEXT_PATH + TICKET + "/book",
    "PAYMENT_RESULT"    : CONTEXT_PATH + TICKET + "/paymentResult",
    "SCREEN_HOME"       : CONTEXT_PATH + ADMIN + "/screen/home",
    "SCREEN_ADD"        : CONTEXT_PATH + ADMIN + "/screen/add",
    "SCREEN_DELETE"     : CONTEXT_PATH + ADMIN + "/screen/delete",
    "SCREEN_SCH_HOME"   : CONTEXT_PATH + ADMIN + "/screenSch/home",
    "SCREEN_SCH_ADD"    : CONTEXT_PATH + ADMIN + "/screenSch/add",
    "SCREEN_SCH_DELETE" : CONTEXT_PATH + ADMIN + "/screenSch/delete"
};

export default urlPaths;

File: movie/src/main/java/kr/com/yh/lotte/UrlPaths.java
package kr.com.yh.lotte;

public class UrlPaths {
    public static final String LOTTE             = "/lotte";

    public static final String LOGIN             = "/login";

    public static final String LOGOUT            = "/logout";

    public static final String SIGNUP            = "/signUp";

    public static final String SIGNUP_SUCCESS    = SIGNUP + "/success";

    public static final String CHECK_EMAIL       = SIGNUP + "/checkEmail";

    public static final String CHECK_ID          = SIGNUP + "/checkId";

    public static final String CHECK_PHONE       = SIGNUP + "/checkPhone";

    public static final String TICKET            = "/ticketing";

    public static final String PERSON_SEAT       = TICKET + "/personSeat";

    public static final String ORDER_SETTLEMENT  = TICKET + "/orderSettlement";

    public static final String BOOK              = TICKET + "/book";

    public static final String PAYMENT_RESULT    = TICKET + "/paymentResult";

    public static final String ADMIN             = "/admin";

    public static final String MEM_HOME          = ADMIN + "/mem/home";

    public static final String MEM_DELETE        = ADMIN + "/mem/delete";

    public static final String MEM_DETAIL        = ADMIN + "/mem/detail";

    public static final String MEM_MODIFY        = ADMIN + "/mem/modify";

    public static final String MEM_FIND          = ADMIN + "/mem/find";
    public static final String SCREEN_HOME       = ADMIN + "/screen/home";
    public static final String SCREEN_ADD        = ADMIN + "/screen/add";
    public static final String SCREEN_DELETE     = ADMIN + "/screen/delete";
    public static final String SCREEN_SCH_HOME   = ADMIN + "/screenSch/home";
    public static final String SCREEN_SCH_ADD    = ADMIN + "/screenSch/add";
    public static final String SCREEN_SCH_DELETE = ADMIN + "/screenSch/delete";
}
